internal/web3/contract_login.go

```diff
@@ -0,0 +1,212 @@
+// Code generated - DO NOT EDIT.
```

Comment:
We should add any code generation steps to the repo so someone else can generate the file if needed in the future. This could be something done in Makefile or simply instructions in the README.

Reply:
We have done this. Good note.